includes/elements/section.php

```diff
@@ -552,6 +552,22 @@ protected function register_controls() {
 				'fields_options' => [
 					'background' => [
 						'frontend_available' => true,
+						'assets' => [
```

Comment:
I don't think this is working. 

Has it been tested?

Why don't we have the same implementation as the container.php?

Reply:
You can test the Sections, by deactivating the Flex Container experiment.